### DISCUSSION TITLE
[1276760]DRAGON BALL : THE BREAKER 七龙珠破界斗士 [本体]

### DISCUSSION THREAD

**Discussion by miaor123** [pjy612/SteamManifestCache#473] [Share]
`appid:1276760`
keys:
```
1276761;b55d19c1067bef37b1163e7ee075621199e867ed2ecdad10961d829fe48ffe08
```

[1276760.zip](https://github.com/pjy612/SteamManifestCache/files/14208497/1276760.zip)

SAI找不到了因此没有CODE。。
邮箱：2814589496@qq.com

**Comment by pjy612** [comment]
* 麻烦 按照格式 #26

**Reply by pjy612** [reply]
* 密钥 或 密钥id 错误 https://steamdb.info/app/1276760/depots/

**Reply by miaor123** [reply]
已修改
